### ISSUE TITLE
Fresh install of 'simple_ostinato' fails with newer version of 'protobuf'

### ISSUE THREAD

**Issue by bdlamprecht** [little-dude/simple-ostinato#15]
I installed 'simple_ostinato' on a new Ubuntu 16.04 VM and tried to import the library into the python interpreter and received the following error:

```
$ python
Python 2.7.12 (default, Nov 19 2016, 06:48:10)
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import simple_ostinato
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/simple_ostinato/__init__.py", line 1, in <module>
    from .stream import Stream
  File "/usr/local/lib/python2.7/dist-packages/simple_ostinato/stream.py", line 8, in <module>
    from . import protocols
  File "/usr/local/lib/python2.7/dist-packages/simple_ostinato/protocols/__init__.py", line 2, in <module>
    from .overrides import MacAddress, Mac, Ethernet, IPv4Address, IPv4, Payload, \
  File "/usr/local/lib/python2.7/dist-packages/simple_ostinato/protocols/overrides.py", line 3, in <module>
    from . import autogenerates
  File "/usr/local/lib/python2.7/dist-packages/simple_ostinato/protocols/autogenerates.py", line 1, in <module>
    from ostinato.protocols import arp_pb2, gmp_pb2, ip4over6_pb2, mld_pb2, \
  File "/usr/local/lib/python2.7/dist-packages/ostinato/protocols/dot2llc_pb2.py", line 13, in <module>
    serialized_pb='\n\rdot2llc.proto\x12\x08OstProto\x1a\x0eprotocol.proto\x1a\ndot3.proto\x1a\tllc.proto\"\t\n\x07\x44ot2Llc:7\n\x07\x64ot2Llc\x12\x12.OstProto.Protocol\x18\xce\x01 \x01(\x0b\x32\x11.OstProto.Dot2Llc')
  File "/usr/local/lib/python2.7/dist-packages/google/protobuf/descriptor.py", line 824, in __new__
    return _message.default_pool.AddSerializedFile(serialized_pb)
TypeError: Couldn't build proto file into descriptor pool!
Invalid proto descriptor for file "dot2llc.proto":
  dot2llc.proto: Import "dot3.proto" has not been loaded.
  dot2llc.proto: Import "llc.proto" has not been loaded.
```

I'm not exactly what the issue is, however, I'm thinking it may have to do with an update that was done to `protobuf`.

**Comment by little-dude** [comment]
Yeah I didn't update `simple-ostinato` for 0.8. Let me do that

**Comment by little-dude** [comment]
@bdlamprecht  it should be fixed in 0.0.4.
